Update Life cycle section with information from https://www.suse.com/de-de/support/policy-products/

cwickert opened the pull request:
This is an attempt to merge the information from https://www.suse.com/c/suse-public-cloud-image-life-cycle/#cloud with https://www.suse.com/de-de/support/policy-products/#cloud

Some information is still subject to review as there are notable differences between the two documents. For example, the blog post says that deprecated images are no longer refreshed while the support policy document claims that do for important security vulnerabilities. @kvanderveer, please assign this to somebody from your team for a _careful_ review. It'll probably take a few iterations to get everything right.

This should also address: 

* https://bugzilla.suse.com/show_bug.cgi?id=1191845
* https://bugzilla.suse.com/show_bug.cgi?id=1191846
* https://bugzilla.suse.com/show_bug.cgi?id=1191847

cwickert committed:
Update Life cycle section with information from
https://www.suse.com/de-de/support/policy-products/#cloud

WIP, some information is still subject to review.

kvanderveer commented:
@cwickert Is there a way I can just view the page with your changes and compare it with what I know as a first review? (Sorry, git novice here).

rjschwei commented on xml/cha_images.xml:
```diff
@@ -470,55 +470,101 @@
+  <itemizedlist>
+   <listitem>
+    <para>
+     Images are refreshed every three months. The new images replace the old
```
s/Images/Active images/

rjschwei commented on xml/cha_images.xml:
```diff
@@ -470,55 +470,101 @@
+    <para>
+     Images are refreshed every three months. The new images replace the old
+     images in the exact state they were in. Replaced images are immediately
+     moving to the <literal>deleted</literal> state.
```
s/deleted/deprecated/

cwickert replied:
Thanks, this is wrong on https://www.suse.com/de-de/support/policy-products/#cloud, too.

kvanderveer replied:
I'm on it

rjschwei commented on xml/cha_images.xml:
```diff
@@ -470,55 +470,101 @@
+   <listitem>
+    <para>
+     If a security vulnerability occurs during the three months that needs to be
+     addressed, all affected images are updated immediately and the three month
```
s/affected images/affected images in active and inactive states/
s/immediately/as soon as possible once the fix for the effected code is available/

rjschwei commented on xml/cha_images.xml:
```diff
@@ -470,55 +470,101 @@
     <listitem>
      <para>
-      Inactive images will only get refreshed for critical security updates.
+      Inactive images are supported unter the rules of LTSS and will only get
```
s/unter/following/
s/LTSS/LTSS or ESPOS/

rjschwei commented on xml/cha_images.xml:
```diff
@@ -470,55 +470,101 @@
-      Deprecated images do not get refreshed. Images in the deprecated state
-      have entered the last six months of their life-cycle and are subject to
-      removal at the end of the six months period.
+      Deprecated images are no longer supported as they have entered the last
```
This is where it gets complicated because the deletion policy is not consistent across providers. Here is my suggestion:
"""
Deprecated images may no longer be supported. The status of support depends on the support status of the product in the image. Deprecated images are not made available in regions added after an image has been set to deprecated and images do no longer get refreshed. At the end of the six month deprecation period images are subject to deletion. It is strongly discouraged to use images in the deprecated state for the creation of new instances.
"""

cwickert replied:
Could we simplify this to
`Deprecated images are only supported if the product in the image is still supported.` (and then link to https://www.suse.com/lifecycle)

rjschwei commented on xml/cha_images.xml:
```diff
@@ -470,55 +470,101 @@
     <listitem>
      <para>
-      Deleted images are no longer available for new instances.
+      Deleted images are no longer supported or available for new instances.
```
s/instances/instance creation/

rjschwei commented:
I didn't pull the repo to read the complete text. Do we have a section that addresses running instances and talks about migration?

cwickert commented:
@kvanderveer Click on the "[Files changed](https://github.com/SUSE/doc-public-cloud/pull/3/files)" tab above this discussion. Red (-) text is old, greeen (+) is new.

Some notes on your bugs

* To address [BSC#1191845](https://bugzilla.suse.com/show_bug.cgi?id=1191845), I included the CVSS rating >+ 9.0
* To mention the BYOS images [BSC#1191846](https://bugzilla.suse.com/show_bug.cgi?id=1191846), I no longer say that on-demand images get refreshed but use "all affected images" instead.
* For [BSC#1191847](https://bugzilla.suse.com/show_bug.cgi?id=1191847), I split the info on release cycle (3 months refresh) and on status (active, inactive, deprecated, deleted). First I explain the general 3 months cycle, then the states.

cwickert committed:
Incorporate review feedback from @rjschwei

cwickert commented:
Thanks @rjschwei, I incorporated your feedback (with some minor changes). Please re-check the comments not yet marked as resolved.

kvanderveer commented on xml/cha_images.xml:
```diff
@@ -470,55 +470,101 @@
+    <para>
+     Images are refreshed every three months. The new images replace the old
+     images in the exact state they were in. Replaced images are immediately
+     moving to the <literal>deleted</literal> state.
```
moving to the <literal>deleted</literal> state.
s/moving/moved

cwickert committed:
Incorporate review feedback from @kvanderveer

cwickert commented:
AFAICS I have addressed all comments here. Is there anything left to do?

> Do we have a section that addresses running instances and talks about migration?

We don't. Do you think there is something to be said about this? AFAICS it's not different from updates on bare metal or VMs.

cwickert commented:
@kvanderveer, did you get my latest message?

karenvdv commented:
No. And I couldn’t find anything wrong with my GitHub settings either. I’m getting messages from other projects I watch. Very strange.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Christoph Wickert ***@***.***>
Sent: Tuesday, December 7, 2021 8:52:27 AM
To: SUSE/doc-public-cloud ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [SUSE/doc-public-cloud] Update Life cycle section with information from https://www.suse.com/de-de/support/policy-products/ (PR #3)


@kvanderveer<https://github.com/kvanderveer>, did you get my latest message?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/SUSE/doc-public-cloud/pull/3#issuecomment-988102066>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AD32LSE7TLO3HKRM3M65ZILUPY3UXANCNFSM5IG6B2LQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

cwickert commented:
I talked to Karen yesterday and will merge now, this PR has been lingering about for too long.
